contributing.md

```diff
@@ -98,16 +101,13 @@ We use the [semantic versioning scheme](https://semver.org/). The semver implies
 
 When incrementing the version number, the following tasks are required:
 
-- Update the version in `gpflow/_version.py`
-- Update the version in the `doc/source/conf.py`
+- Update the version in `./VERSION`
+- Update the version in `./doc/source/conf.py`
 - Add a note to `RELEASE.md`
 
 ### Keeping up with TensorFlow
 
 GPflow tries to keep up with API changes in TensorFlow as far as is reasonable, so that the latest GPflow will work with the latest stable TensorFlow release. Changing the version of TensorFlow that we're compatible with requires a few tasks:
 
-- Update version used on travis via `travis.yml`
-- Update version used on codeship (requires codeship login)
 - Update `README.md`
-- Update version used by readthedocs.org via `docsrequire.txt`
-- Increment the GPflow version (see below).
+- Increment the GPflow version (see above).
```

Comment:
Does anything else need to happen with the current setup? I think both CircleCI and readthedocs automatically pull in the latest tensorflow ?

Reply:
@awav ?

Reply:
Yes, they are pulling the latest version.